subsys/bluetooth/audio/Kconfig.tbs

```diff
@@ -0,0 +1,212 @@
+# Bluetooth Audio - Call control configuration options
+#
+# Copyright (c) 2020 Bose Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if BT_AUDIO
+
+#####################  Telephone Bearer Service #####################
+
+config BT_TBS
+	bool "Telephone Bearer Service Support"
+	select BT_CCID
+	help
+	  This option enables support for Telephone Bearer Service.
+
+if BT_TBS
+
+config BT_GTBS
+	bool "Generic Telephone Bearer Service Support"
+	default y
+	help
+	  This option enables support for Generic Telephone Bearer Service.
+
+config BT_TBS_PROVIDER_NAME
+	string "Telephone Bearer Service Provider Name"
+	default "Unknown"
+	help
+	  Sets the name of the service provider for the bearer.
+
+config BT_TBS_UCI
+	string "Telephone Bearer Service Uniform Caller Identifier (UCI)"
+	default "un000"
+	help
+	  Sets the UCI of the bearer. See
+		https://www.bluetooth.com/specifications/assigned-numbers/uniform-caller-identifiers/
+		for a table of valid UCIs.
+
+config BT_TBS_TECHNOLOGY
+	int "Telephone Bearer Service Technology"
+	range 1 10
+	help
+	  Sets the technology used for the bearer, e.g. GSM, LTE and 5G.
+	  1 : 3G
+	  2 : 4G
+	  3 : LTE
+	  4 : Wi-Fi
+	  5 : 5G
+	  6 : GSM
+	  7 : CDMA
+	  8 : 2G
+	  9 : WCDMA
+	  10: IP
```

Comment:
Agreed. The idea is definitely also to put these into runtime registrations as we did for e.g. the BT_VCS implementation. However, that will need to be fixed in a later PR, as the idea of this, as with any of the other LE Audio services, is that we simply upstream what we have, so that it is publicly available.

Reply:
Would a choice be clearer to users so it's obvious what they are selecting / what is selected ?

In `menuconfig`:
```
Telephone Bearer Service Technology (Wi-Fi)
```
and:
```
( ) 3G
( ) 4G
( ) LTE
(X) Wi-Fi
```

and then in `prj.conf`:
```
CONFIG_BT_TBS_TECHNOLOGY_WIFI=y
```
vs:
```
CONFIG_BT_TBS_TECHNOLOGY=4 # <-- What does this mean, damn I need to look at the docs :(
```

Reply:
@tejlmand yeah, that sounds like a better approach, at least as long as this comes from Kconfig

Reply:
I'd rather not start modifying things that aren't broken when simply upstreaming an existing implementation. The next step for this would be to dynamically register the TBS instances, which would solve this.